### PR TITLE
feat: shopware 6.5

### PR DESCRIPTION
**Pull request by stefanpoensgen**

### CODE REVIEW

**Comment by tinect** [comment]
Thank you. Could you please remove `height` from the thumbnailUrlTemplate?

File: src/Storefront/Framework/Twig/Extension/UrlEncodingTwigFilter.php [hunk 1]
```diff
@@ -14,7 +13,7 @@ class UrlEncodingTwigFilter extends AbstractExtension
     public function getFilters(): array
     {
         return [
-            new TwigFilter('frosh_encode_url', [$this, 'encodeUrl']),
+            new TwigFilter('frosh_encode_url', fn (?string $mediaUrl): ?string => $this->encodeUrl($mediaUrl)),
```
Comment (shyim) [inline src/Storefront/Framework/Twig/Extension/UrlEncodingTwigFilter.php:16]:
```suggestion
            new TwigFilter('frosh_encode_url', $this->encodeUrl(...)),
```

**Review by tinect — CHANGES_REQUESTED** [review]
@stefanpoensgen Could you please verify the changes in shopwares ThumbnailService?

EDIT: done

File: composer.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
         }
     ],
     "require": {
-        "shopware/core": "^6.4.10"
+        "shopware/core": "^6.5.0"
```
Comment (tinect) [inline composer.json:24]:
```suggestion
        "shopware/core": "~6.5.0"
```

**Comment by tinect** [comment]
- [ ] use PhpParser? we just need to overwrite the private method `createThumbnailsForSizes` and set `$strict` to false in `updateThumbnails`

**Comment by tinect** [comment]
@stefanpoensgen please test :-)

**Review by tinect — APPROVED** [review]

**Comment by tinect** [comment]
- [ ] theme:compile broken

**Pull request merged**